app/components/request/configs.js

```diff
@@ -44,6 +56,15 @@ export default Component.extend(CanTriggerBuild, TriggerBuild, {
     }
   },
 
+  onCustomize() {
+    if (this.status == 'customize') {
```

Comment:
I'd suggest using `customizing` computed property here:
```suggestion
    if (this.customizing) {
```

Reply:
yep! this has already happened at some later point: https://github.com/travis-ci/travis-web/blob/sf-request-config-tweaks/app/components/request/configs.js#L64